### PR TITLE
[onesignal]Update onesignal.rb to include optional organization_id

### PR DESCRIPTION
**Pull request by brandon-carpenter**
🔑 adds optional, organization_id, paramater to onesignal.rb

### Checklist
- [x] I've run `bundle exec rspec` from the root directory to see all new and existing tests pass
- [x] I've followed the _fastlane_ code style and run `bundle exec rubocop -a` to ensure the code style is valid
- [x] I've read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I've updated the documentation if necessary.

### Motivation and Context
This change adds the [Onesignal](https://documentation.onesignal.com/reference/create-an-app) organization_id as an optional parameter.  Presently, when new Onesignal applications are created, they are created without being assigned to an organization.  This allows end users to specify an organization_id if desired.  Apps created under an organization will delegate the Onesignal org-level-admins to that application.  Org-level-admins are unable to view/modify apps not within the organization.

No issue has been created.  Please let me know if you'd like an issue created for this PR.

### Description
The additional parameter was added inline with the other parameters in onesignal.rb.  Payload has been updated along with self.available_options, and self.example_code.  

Both `bundle exec rubocop -a` and `bundle exec rspec` as referenced [here](https://github.com/fastlane/fastlane/blob/master/Testing.md) were ran following the changes.
  

### Testing Steps
`bundle exec onesignal` was ran with a Fastfile containing the necessary parameters including `organization_id`.   This resulted in a new application being created within Onesignal under the provided organization_id. This allowed org-level-admins to view the newly created onesignal application.  Additionally,  the changes were ran on an app update successfully, and on an app without the organization_id provided.  On the latter, the app was created without being assigned to an organization. The changes will also update an application that was created outside of an organization and will move it under the provided organization.

### CODE REVIEW

**Comment by brandon-carpenter** [comment]
@googlebot I signed it!

File: fastlane/lib/fastlane/actions/onesignal.rb [hunk 1]
```diff
@@ -17,6 +17,7 @@ def self.run(params)
         apns_p12_password = params[:apns_p12_password]
         android_token = params[:android_token]
         android_gcm_sender_id = params[:android_gcm_sender_id]
+        organization_id = params[:organization_id].to_s.strip
```
Comment (joshdholtz) [inline fastlane/lib/fastlane/actions/onesignal.rb:20]: Calling `.to_s.strip` would never allow `organization_id` to be `nil` so I think we need to remove this 😊 

```suggestion
        organization_id = params[:organization_id]
```

File: fastlane/lib/fastlane/actions/onesignal.rb [hunk 2]
```diff
@@ -135,7 +137,13 @@ def self.available_options
                                        env_name: "APNS_ENV",
                                        description: "APNS environment",
                                        optional: true,
-                                       default_value: 'production')
+                                       default_value: 'production'),
+
+          FastlaneCore::ConfigItem.new(key: :organization_id,
+                                       env_name: "ORGANIZATION_ID",
```
Comment (joshdholtz) [inline fastlane/lib/fastlane/actions/onesignal.rb:143]:
```suggestion
                                       env_name: "ONE_SIGNAL_ORGANIZATION_ID",
```

File: fastlane/lib/fastlane/actions/onesignal.rb [hunk 3]
```diff
@@ -135,7 +137,13 @@ def self.available_options
...
+                                       sensitive: true,
+                                       description: "ONESIGNAL ORGANIZATION ID",
```
Comment (joshdholtz) [inline fastlane/lib/fastlane/actions/onesignal.rb:145]:
```suggestion
                                       description: "OneSignal Organization ID",
```

**Review by joshdholtz — CHANGES_REQUESTED** [review]
This looks good! Just a few small requests 😊

**Comment by brandon-carpenter** [comment]
Updated with your suggestions.  Thanks!

**Review by joshdholtz — APPROVED** [review]
Looks great! Thank you so much for the contribution ❤

**Pull request merged**